spec.html

```diff
@@ -39495,6 +39245,32 @@ <h1>GeneratorYield ( _iterNextObj_ )</h1>
           1. NOTE: This returns to the evaluation of the operation that had most previously resumed evaluation of _genContext_.
         </emu-alg>
       </emu-clause>
+
+      <emu-clause id="sec-yield" aoid="Yield">
+        <h1>Yield ( _value_ )</h1>
+        <p>The abstract operation Yield takes argument _value_. It performs the following steps when called:</p>
+        <emu-alg>
+          1. Let _generatorKind_ be ! GetGeneratorKind().
+          1. If _generatorKind_ is ~async~, then return ? AsyncGeneratorYield(_value_).
+          1. Otherwise, return ? GeneratorYield(CreateIterResultObject(_value_, *false*)).
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-createiteratorfromclosure" aoid="CreateIteratorFromClosure">
+        <h1>CreateIteratorFromClosure ( _closure_, _generatorBrand_, _generatorPrototype_ [ , _generatorSlots_ ] )</h1>
+        <p>The abstract operation CreateIteratorFromClosure takes arguments _closure_ (an Abstract Closure with no parameter), _generatorBrand_, _generatorPrototype_ (an Object) and optional argument _generatorSlots_ (a List of names of internal slots). It performs the following steps when called:</p>
+        <emu-alg>
+          1. Assert: _closure_ is an Abstract Closure.
+          1. 1. NOTE: _closure_ can contain uses of the Yield shorthand to yield an IteratorResult object.
```

Comment:
```suggestion
          1. NOTE: _closure_ can contain uses of the Yield shorthand to yield an IteratorResult object.
```
altho this is probably better in an emu-note, not in the algorithm steps